mailbox/api/src/main/java/org/apache/james/mailbox/model/Cid.java

```diff
@@ -47,6 +48,16 @@ private static String unwrap(String cidAsString) {
         return unwrapCid;
     }
 
+    public static Optional<Cid> fromRelaxedNoUnwrap(String cidAsString) {
```

Comment:
This comment is unrelated to this PR and deserve to be here: https://github.com/linagora/james-project/pull/921

(However, it is important to not remove table migration before being able to migrate ALL old entries, hence I put this branch on top of the other one to not forget nor do mistakes while merging.)